stdlib/public/Concurrency/linker-support/magic-symbols-for-install-name.c

```diff
@@ -34,6 +34,11 @@
 // Xcode inserting a runpath search path of /usr/lib/swift based on the deployment target being less than
 // SupportedTargets[target][SwiftConcurrencyMinimumDeploymentTarget] in SDKSettings.plist.
 
+// Note: even though the embedded Concurrency library is only supported back to
```

Comment:
// Clients can back deploy to OS versions that predate Concurrency as an embedded library, and conditionally use
// it behind an #availability check. Such clients will still need to link the embedded library instead of the OS version.
// To support that, set the start version to Swift's first supported versions: macOS (née OS X) 10.9, iOS 7.0,
// watchOS 2.0, tvOS 9.0 rather than Concurrency's first supported versions listed above.